src/apps/statements/tests/storingStatements/queriables/getMetadataFromStatement/getDurationMetadata.test.ts

```diff
@@ -0,0 +1,37 @@
+import * as assert from 'assert';
+import { assign } from 'lodash';
+
+import { getDurationMetadata } from '../../../../service/storeStatements/queriables/getMetadataFromStatement/getDurationMetadata';
+import { statementDefaults } from './fixtures/statements.fixture';
+
+describe(
+  'Retrieve duration metadata from statement',
+  () => {
```

Comment:
For consistency with other tests, probably better to put this on one line if it doesn't exceed the line length limit in the linting rules. There are some other occurrences of this pattern in this PR that I think should also be changed for consistency.

Reply:
Well.. this is exactly why I did so 🤔  You have a 100max lines lint rule per file

Reply:
Got it 👍